### ISSUE TITLE
Improve presentation of full activity list. 

### ISSUE THREAD

**Issue by chtowner** [BathHacked/energy-sparks#249]
Activity Categories List – Current Presentation is poor. Small blue writing difficult to read. Long list is overwhelming. Grouping under different categories is not intuitive so difficult to find an activity again. Children suggested we could show a sub-set of activities to participants first – perhaps those that are easier to achieve and earn you 5 points, and then as schools achieve those, the  site could reveal more activities with higher numbers of points.

**Comment by ldodds** [comment]
I've reorganised the page so that it groups together the activities into sections to make it easier to navigate. The font is also larger and I've sorted the activities so lowest scoring appears first. There's now a lot less on the page to take in.

Every activity category now also has a description, which can be edited by an administrator. I thought it would be useful to explore what that category is about, to help users understand the types of activity it covers.

Thinking that through might also help us organise the list if there's some that don't fit correctly in one category or another.

For the recommendation aspect, I'm going to address this as part of #248
